### PR TITLE
[Localization] Fix some Chinese translations

### PR DESCRIPTION
**Pull request by ghost**
Change Logs:

1) `其他` to `其它`。
2) Remove useless spaces, and change English comma to Chinese one.
3) `Sign up for` doesn't mean `注册`. It means `报名登记`、`参与……`. According to the context, `注册订阅` is a right meaning.

### CODE REVIEW

File: locale/zh-cn/index.md [hunk 1]
```diff
@@ -5,9 +5,9 @@ labels:
   current-version: 当前版本
   download: 下载
   download-for: 下载为
-  other-downloads: 其他下载
-  other-lts-downloads: 其他 LTS 下载
-  other-current-downloads: 其他最新版本下载
+  other-downloads: 其它下载
+  other-lts-downloads: 其它LTS版下载
```
Comment (laosb) [inline locale/zh-cn/index.md]: Make sure you have spaces between CJK and ASCII chararcters.

File: locale/zh-cn/index.md [hunk 2]
```diff
@@ -17,8 +17,8 @@ labels:
   version-schedule-prompt: 可参考
   version-schedule-prompt-link-text: LTS 日程。
   newsletter: true
-  newsletter-prefix: 注册
-  newsletter-postfix: ", Node.js 官方的新闻周报。"
+  newsletter-prefix: 注册订阅
```
Comment (laosb) [inline locale/zh-cn/index.md]: Actually you may just remove 注册.

**Review by laosb — CHANGES_REQUESTED** [review]

**Comment by laosb** [comment]
Good job, thank you!

File: locale/zh-cn/index.md (hunk 1 above)
Comment (ghost) [reply locale/zh-cn/index.md]: OK, fixed.
PS: Why? The formation recognization?

File: locale/zh-cn/index.md (hunk 2 above)
Comment (ghost) [reply locale/zh-cn/index.md]: OK. Fixed

**Comment by ghost** [comment]
You're welcome! All things have all been fixed.

**Review by amorist — APPROVED** [review]

**Review by laosb — APPROVED** [review]
LGTM!

**Pull request merged**